sql/core/src/test/resources/sql-tests/results/postgreSQL/date.sql.out

```diff
@@ -685,12 +685,12 @@ select make_date(2013, 2, 30)
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.SparkDateTimeException
+org.apache.spark.SparkException
 {
-  "errorClass" : "_LEGACY_ERROR_TEMP_2000",
+  "errorClass" : "INTERNAL_ERROR",
+  "sqlState" : "XX000",
   "messageParameters" : {
-    "ansiConfig" : "\"spark.sql.ansi.enabled\"",
-    "message" : "Invalid date 'FEBRUARY 30'"
+    "message" : "The Spark SQL phase optimization failed with an internal error. You hit a bug in Spark or the Spark plugins you use. Please, report this bug to the corresponding communities or vendors, and provide the full stack trace."
```

Comment:
Clearly this is not an internal error, given you just provoked it with SQL.

Reply:
Ah, okay I think we need separate rule to extract JAVA error message that started with "Invalid date"